spec/features/dashboard_spec.rb

```diff
@@ -46,15 +46,13 @@
     end
 
     scenario 'Multiple selection through sheets' do
-      visit '/dashboard/2018/02'
+      visit '/dashboard/2018/01'
 
       find('td.inner', text: '19').click
       find('td.inner', text: '20').click
       expect(page).to have_content('Selecionado (2)')
-      expect(page).to have_content('Selecionado (2)')
-
 
-      within 'tbody > tr:nth-child(1)' do
+      within 'tbody > tr:nth-child(3)' do
```

Comment:
Question: why this number has changed? (I don't know the answer, I am just curious about that hahaha)

Reply:
Before, the 15th day of the month was the base day. So I [changed](https://github.com/Codeminer42/Punchclock/pull/168/files#diff-1358ec85630c58187bcdc2a3760746bab34b1737b015cde67dd1ec2cebd7da52R5) the base day to the 1st. This caused the move of the week lines.

Reply:
Got it, thanks!